cli/medperf/logging/formatters/newline_formatter.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+
+class NewLineFormatter(logging.Formatter):
+    def __init__(self, fmt, datefmt=None):
+        """
+        Init given the log line format and date format
+        """
+        logging.Formatter.__init__(self, fmt, datefmt)
+
+    def format(self, record):
+        """
+        Override format function
+        """
+        msg = logging.Formatter.format(self, record)
+
+        if record.message != "":
+            parts = msg.split(record.message)
+            msg = msg.replace("\n", "\n" + parts[0])
```

Comment:
what does this do (I am a bit confused about splitting using the record.message)

Reply:
I'm adding comments to explain this. Basically we're separating the logging prefix (timestamp | section_of_code_calling_logging | loglevel) from the actual message so that it can applied to each newline.

This was necessary as these sections of logging were breaking the structure of the logging file due to newlines